### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by fullsushidev**
Introducing bug report and pull request template.

### CODE REVIEW

**Comment by marcomicera** [comment]
I personally like using this https://github.com/stevemao/github-issue-templates/tree/master/simple, I find it more concise, but it's just personal taste...

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Additional context**
+Add any other context about the problem here.
```
Comment (marcomicera) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: Perhaps the Docker image tags being used?

Comment (fullsushidev) [reply .github/ISSUE_TEMPLATE/bug_report.md:27]: We don't really have it right now. I'd suggest to change it again as soon as we finish this new setup with k8s.

**Comment by fullsushidev** [comment]
> I personally like using this https://github.com/stevemao/github-issue-templates/tree/master/simple, I find it more concise, but it's just personal taste...

Did you mean about both files? The issue template seems to be the same, just different order/wording!?
And the PR template is way simpler, but for now, I'd say that we need more details since we're lacking documentation.

What do you think about moving on as is, and make it simpler (if we can) within time? @marcomicera

**Comment by marcomicera** [comment]
> What do you think about moving on as is, and make it simpler (if we can) within time?

Sure

**Pull request merged**